src/extension.ts

```diff
@@ -232,6 +232,7 @@ async function stopClients(folders = prunedFolderNames()) {
 		if (clients.has(folder)) {
 			promises.push(clients.get(folder).client.stop());
 			clients.delete(folder);
+			reporter.sendTelemetryEvent('stopClient');
```

Comment:
There is probably some complexity cost to weigh this against, but is it worth tracking the context? i.e. pass an extra argument to `stopClients` which (if sent as a 2nd argument of `sendTelemetryEvent`) can tell us whether this was manually requested stop, or how was it triggered?

Reply:
I'd be worried about creating a lot of noise that wouldn't tell us all that much. I think we'll get more out of logging the command execution (and I could add that now, it's just one more line).

Reply:
Fair enough 👍🏻